File: app/src/main/java/ru/cool/lwjgl/input/Buttons.java
package ru.cool.lwjgl.input;

import org.lwjgl.glfw.GLFW;

public enum Buttons {
    RIGHT(GLFW.GLFW_KEY_D),
    LEFT(GLFW.GLFW_KEY_A),
    FORWARD(GLFW.GLFW_KEY_W),
    BACK(GLFW.GLFW_KEY_S),
    UP(GLFW.GLFW_KEY_SPACE),
    DOWN(GLFW.GLFW_KEY_LEFT_CONTROL),
    ESCAPE(GLFW.GLFW_KEY_ESCAPE);

    private final int glfwKey;
    Buttons(int glfwKey){
        this.glfwKey = glfwKey;
    }

    public int getGlfwKey(){
        return glfwKey;
    }
}

File: kotlin_app/src/main/kotlin/ru/cool/lwjgl_kotlin/input/Buttons.kt
package ru.cool.lwjgl_kotlin.input

import org.lwjgl.glfw.GLFW

enum class Buttons(keyId: Int) {
    ESCAPE(GLFW.GLFW_KEY_ESCAPE),
    W(GLFW.GLFW_KEY_W),
    A(GLFW.GLFW_KEY_A),
    S(GLFW.GLFW_KEY_S),
    D(GLFW.GLFW_KEY_D),
    R(GLFW.GLFW_KEY_R),
    F(GLFW.GLFW_KEY_F),
    SPACE(GLFW.GLFW_KEY_SPACE),
    LEFT_CTR(GLFW.GLFW_KEY_LEFT_CONTROL);

    var keyId = keyId

}